addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1312,6 +1312,15 @@ def _load_more_members(self, known_member_ids):
         )
         return Store(unknown_members).add(self, "member_count").get_result()
 
+    def _clear_recipients_on_empty_message(self, msg_values):
```

Comment:
This function name is also not ideal, as this override does something else than clearing recipients.

Maybe `_clean_empty_message`?